### ISSUE TITLE
Missing indication for external control

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3109]
The "Open Tutorials" button opens content in a browser window, but there is no visual  or programmatic indication to warn users of this action.  
Suggested Solution 
Ensure that all users are warned about the functionality of the "Open Tutorials" control: 

- Provide an icon positioned next to the control's text, inside the boundaries of the  menuitem so that it is clickable and encompassed by the focus outline. The icon must have a color contrast ratio of 3:1 with its background and be hidden from  screen reader users (via aria-hidden="true" on an SVG or empty alt attribute value  on an <img>). 
- Provide a text alternative for the icon as off-screen text inside the control, for screen  reader users. This should be implemented as a span with the appropriate screen reader-only class, with the text "(opens in new tab)" in parentheses. There should  also be a non-breaking space before the left parenthesis, so that it does not become adjoined to the control's name. 
- Optionally, when the link receives focus and mouse hover, you can show the off screen text visually too (i.e. temporarily remove the sr-only class), as a sort of accessible tooltip and additional reinforcement. This can be helpful but may not fit  into the design. 

Below is the icon we recommend using for opening new tabs: 

![207937976-9d80c423-12b7-4193-b67c-9f81d20125a5](https://github.com/OpenSpace/OpenSpace/assets/99894471/3afc12d2-ff00-4dfe-99ac-107b00f42ba5)

**Comment by alexanderbock** [comment]
@ylvaselling   We didn't do this one yet

**Comment by WeirdRubberDuck** [comment]
Could maybe add a dimmed version of this icon to the right section in the help menu items that open in browser? 

![Image](https://github.com/user-attachments/assets/a78b6eaf-3833-4d25-aab0-7abadf367697)

Or just replace the left icon

**Comment by alexanderbock** [comment]
Dimmed version on the right section would be
